Ref/README.md

```diff
@@ -26,22 +26,19 @@ fprime-util generate
 Now that the build directory has been generated, the user need not run `fprime-util generate` again unless the build directory has been removed.
 
 The next step is to build the Ref application's code. This is done for the current system that the user is running on. This is handled by CMake
-and will produce a binary that can be run on the user's system. This is accomplished by using the `build` subcommand of `fprime-util`. Since we
-will run the code next, we will also run the install command to ensure we may easily find the binaries.
-
-```
-fprime-util install
-```
+and will produce a binary that can be run on the user's system. This is accomplished by using the `build` subcommand of `fprime-util`.
 
 ## Running the F´ Ground System and Code
 
-F´ ships with a browser-based test ground system. This system is designed to help developers of F´ projects quickly test and work with F´ code
-without much overhead. This ground system can be run with the following commands. Please note: the Ref application's binary will also be run
-automatically.  This allows for quick testing on Linux and Mac OSX. Please ensure that the binary has been built and installed as described above.
+F´ ships with a browser-based test ground system. This system is designed to help developers of F´
+projects quickly test and work with F´ code without much overhead. This ground system can be run
+with the following commands. Please note: the Ref application's binary will also be run
+automatically. This allows for quick testing on Linux and Mac OSX. Before running the GDS, make sure
+that you have built the Ref example.
 
 ```
 cd fprime/Ref
-fprime-gds -d .
+fprime-gds
```

Comment:
Why wouldn't you improve the documentation to run from the packaged output?

Reply:
I figured `fprime-gds` was cleaner than `fprime-gds -r build-artifacts`. Since we added artifact root autodetection we might as well take advantage of it, right?

Reply:
Won't this fail because there is no settings.ini to tell it which artifact directory to use?

Reply:
No, because even if the settings.ini is missing it'll provide the default location `build-artifacts`.

Detection order: `--root` argument, else `settings.ini`, else `build-artifacts`.